Src/Support/Google.Apis.Auth.AspNetCore/GoogleOpenIdConnectDefaults.cs

```diff
@@ -0,0 +1,28 @@
+﻿/*
+Copyright 2018 Google Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Google.Apis.Auth.AspNetCore
+{
+    public static class GoogleOpenIdConnectDefaults
```

Comment:
I assume this needs to be public? If so, we definitely want XML comments for everything.

Reply:
Yes, this needs to be public. XmlDoc added.